src/test/java/org/openmrs/performance/registries/DoctorRegistry.java

```diff
@@ -73,8 +73,11 @@ public ChainBuilder openVitalsAndBiometricsTab(String patientUuid) {
 				MID_UPPER_ARM_CIRCUMFERENCE
 		);
 		return exec(httpService.getPatientObservations(patientUuid, vitals))
-				.exec(httpService.getPatientObservations(patientUuid, biometrics))
-				.exec(httpService.saveVitalsData(patientUuid));
+				.exec(httpService.getPatientObservations(patientUuid, biometrics));
+	}
+
+	public ChainBuilder saveVitals(String patientUuid){
```

Comment:
I think this makes much sense to rename this to `recordVitals`. I mean only the registry method.

```suggestion
	public ChainBuilder recordVitals(String patientUuid){
```